tests/test_abcs_parameters.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python3
+
+###########################################################################
+#
+#    Copyright 2024 Google LLC
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#            https://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+###########################################################################
+
+""" Module to test ABCD parameters """
+
+from utils.utils import build_abcd_params_config
+
+
+class ArgsMock():
```

Comment:
consider [dataclass](https://docs.python.org/3/library/dataclasses.html) here. It will remove the need to add an __init__ method.

Reply:
+1!